LondonBritishLibrary/orient/BLorient657.xml

```diff
@@ -0,0 +1,256 @@
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.1">
+<locus target="3r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Maggabit">Maggābit</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.2">
+<locus target="35v"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Miyazya">Miyāzyā</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.3">
+<locus target="63r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Genbot">Gǝnbot</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.4">
+<locus target="100r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Sane">Sane</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.5">
+<locus target="135r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Hamle">Ḥamle</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.6">
+<locus target="182v"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Nahase">Naḥāse</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.7">
+<locus target="214r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Pagwemen">Ṗagʷǝmen</title>
+<textLang mainLang="gez"/>
+</msItem>
+</msContents>
+
+<physDesc>
+
+<objectDesc>
+  <layoutDesc>
+      <layout columns="3" writtenLines="30 33"/>
+  </layoutDesc>
+</objectDesc>
+
+  <handDesc>
+    <handNote xml:id="h1" script="Ethiopic">
+    <desc>Written in a fine hand, of the <date notBefore="1600" notAfter="1667">17th Cent.</date>, before 1667.</desc>
+    </handNote>
+    </handDesc>
+
+  <decoDesc>
+  <decoNote type="drawing" xml:id="d1">
+  <locus target="#1"/>
+  <desc>A picture in outline.</desc>
+  </decoNote>
+  </decoDesc>
+
+  <additions>
+  <list>
+  <item xml:id="a1">
+  <locus from="2v"/>
+  <desc type="GuestText"><title type="complete">A short chronological section from Adam to Christ, in Amharic.</title></desc>
+  <q xml:lang="am"><seg part="F">ዘንተ፡ ዘአጽሐፈ፡ <persName role="patron" ref="PRS13283SahlaMikael">ሣህለ፡ ሚካኤል፡</persName> <placeName ref="LOC5662Semada">ዘስማዳ</placeName>፡ ወዘጸሐፈ፡
+  <persName role="scribe" ref="PRS13284GabraIyasus">ገብረ፡ ኢየሲስ፡</persName> ዘ<placeName ref="LOC6563Sada">ፀዳ፡</placeName> ዘተቀብረ፡ ቦቱ፡ ጻድቅ፡ <persName ref="PRS10375Yohanne">ዮሐንስ፡</persName> ንጉሠ፡ ነገሥት፡
+  አንቢበክሙ፡ እንዘ፡ ትፌጽሙ፡ ፩አቡነ፡ ዘበሰማያት፡ በእንተ፡ እግዚአብሔር፡ አብ።</seg></q>
+  </item>
+
+  <item xml:id="a2">
+    <locus target="#4r"/>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+    </desc>
+    <q xml:lang="gez">ስንክሳር፡ ዘቅዱስ፡ መድኃኔ፡ ዓለም።</q>
+  </item>
+
+  <item xml:id="a3">
+  <locus target="#2r #220v #221r"/>
+  <desc>Hymns to John the Baptist.</desc>
+  </item>
+
+  <item xml:id="e1">
+  <locus target="#2r"/>
+  <desc>The name of ʾabbā Gabra ʾIyasus is on <locus target="#2r">f. 2a</locus>.</desc>
```

Comment:
Could you, please, create a person record for him. You do not need to write out locus (f. 2a), if it is encoded, the string will be print out automatically.